tests/test_client/tests.py

```diff
@@ -902,6 +903,28 @@ def _generic_view(request):
     return HttpResponse(status=200)
 
 
+urlpatterns = [
+    path('', _generic_view, name='test-client-generic-view')
+]
```

Comment:
Can we move this to a separate file, e.g. `tests/test_client/urls_overridden_urlconf.py`:
```python
def empty_response(request):
    return HttpResponse()


urlpatterns = [
    path('', empty_response, name='overridden_urlconf_view'),
]
```